Tests/Packet++Test/Tests/TcpTests.cpp

```diff
@@ -325,11 +327,18 @@ PTF_TEST_CASE(TcpPacketCreation2)
 
 	PTF_ASSERT_BUF_COMPARE(tcpPacket.getRawPacket()->getRawData(), buffer1, bufferLength1);
 
-	pcpp::TcpOption qsOption = tcpLayer.addTcpOptionAfter(pcpp::TcpOptionBuilder(pcpp::TCPOPT_QS, nullptr, PCPP_TCPOLEN_QS), pcpp::TCPOPT_MSS);
+	pcpp::TcpOption qsOption =
+	    tcpLayer.addTcpOptionAfter(pcpp::TcpOptionBuilder(pcpp::TCPOPT_QS, nullptr, PCPP_TCPOLEN_QS), pcpp::TCPOPT_MSS);
 	PTF_ASSERT_TRUE(qsOption.isNotNull());
 	PTF_ASSERT_TRUE(qsOption.setValue(htobe32(9999)));
-	PTF_ASSERT_TRUE(tcpLayer.addTcpOption(pcpp::TcpOptionBuilder(pcpp::TcpOptionEnumType::Snack, static_cast<uint32_t>(htobe32(1000)))).isNotNull());
-	PTF_ASSERT_TRUE(tcpLayer.insertTcpOptionAfter(pcpp::TcpOptionBuilder(pcpp::TcpOptionBuilder::NopEolOptionEnumType::Nop), pcpp::TcpOptionEnumType::Timestamp).isNotNull());
+	PTF_ASSERT_TRUE(
+	    tcpLayer
+	        .addTcpOption(pcpp::TcpOptionBuilder(pcpp::TcpOptionEnumType::Snack, static_cast<uint32_t>(htobe32(1000))))
+	        .isNotNull());
+	PTF_ASSERT_TRUE(tcpLayer
+	                    .insertTcpOptionAfter(pcpp::TcpOptionBuilder(pcpp::TcpOptionBuilder::NopEolOptionEnumType::Nop),
+	                                          pcpp::TcpOptionEnumType::Timestamp)
+	                    .isNotNull());
```

Comment:
This is not very nice, maybe we should disable it?

Reply:
fixed.

Reply:
Actually the same format appears many times in this file. Do you think we should give up an let clang-format do its thing or add ignores everywhere? 🤔

Reply:
I perfer to just follow what clang-format does if we format a lot of code at once; otherwise there are too many trivial issues. If we are developing normal PR, we can decide whether to disable a certain part of code block.

Reply:
ok, let's just keep it as is. Maybe we can remove `clang-format off` in line 334 to make the entire file consistent

Reply:
Reverted.